src/components/Game.js

```diff
@@ -22,14 +49,14 @@ const Game = () => {
       <p>Please follow the following format for your poetry submission:</p>
 
       <p className="Game__format-example">
-        { exampleFormat }
+        { stringFormat }
```

Comment:
stringFormat is a function here, so you can't render it.  I'm not sure what this is supposed to do.